README.md

```diff
@@ -9,6 +9,8 @@ Check out [the contribution guide](https://github.com/trufflesuite/trufflesuite.
 
 We welcome all contributions, typo corrections, and general feedback. See our [full contribution guide](https://github.com/trufflesuite/trufflesuite.com/blob/master/CONTRIBUTING.md) for details on site architecture and how you could help.
 
+**Please base your changes and target your PR against the `develop` branch, not `master`**
```

Comment:
I think this doesn't need to be bold, since the other changes mean it'll be a much less likely situation